src/dotnet/commands/dotnet-help/HelpCommandParser.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Microsoft.DotNet.Cli.CommandLine;
+
+namespace Microsoft.DotNet.Tools.Help
+{
+    internal static class HelpCommandParser
+    {
+        public static Command Help()
+        {
+            return Create.Command(
+                "help",
+                LocalizableStrings.AppFullName,
+                Accept.ZeroOrOneArgument()
+                    .With(
+                        LocalizableStrings.CommandArgumentDescription,
+                        LocalizableStrings.CommandArgumentName),
+                Create.Option(
+                    "-h|--help|-?|/?",
```

Comment:
We do not currently handle the -?|/? for any of the other commands in the CLI. Do not add it here. We can talk about supporting these extra formats as well, but we decide to do so, we need to do it for all our commands.